[website] New wave of open roles (#35240)
mui/material-ui: docs/src/pages/careers/react-tech-lead-core.md

We've seen rapid growth with all of them (2-3x per year) and we have more to come.
We are building [a low-code platform](https://mui.com/toolpad/) for developers to significantly increase the speed and lower the barrier for creating simple CRUD/dashboard UIs with MUI.

We are a fully distributed team, spread across multiple time zones.
We work mainly asynchronously, relying mostly on written collaboration.
Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
For additional details about the MUI team and culture, you can check our [careers](https://mui.com/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).

## Why we're hiring

The core components team (MUI Core) needs help.
They are working on 4 products (Material UI, MUI Base, Joy UI, and MUI System) with a tiny team.

> We also need help to continue to improve the health of the open-source product: make the core components easier to use, engage more with the community, polish all the details, make the components more accessible, guiding developers to answers, and just generally being a positive presence in the open-source community.

Reviewer: ```suggestion
We also need help to continue to improve the health of the open-source product: make the core components easier to use, engage more with the community, polish all the details, make the components more accessible, guiding developers to answers, and just generally be a positive presence in the open-source community.
```

Both our open-source community and our premium products are growing fast (x2 YoY).
We need talented people to keep that going!

### Why this is interesting

Our solution empowers React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.

The core components are widely used, you will receive a lot of feedback on your work. The community has high expectations, they will push you to do better, every day.

> The [results of the latest survey](https://mui.com/blog/2021-developer-survey-results/) can give you a great sense for where we will need to push the open-source library in the coming months.

Reviewer: ```suggestion
The [results of the latest survey](https://mui.com/blog/2021-developer-survey-results/) can give you a great sense of where we will need to push the open-source library in the coming months.
```

- **Lead**. You will leads by inspiring and setting the bar for code quality for the engineering team, and cracking the hard problems:
  - Guide the approach and execution.
  - Guide people in making the right technical decisions themselves over imposing them.
  - Be aware of the current technical landscape and knows where the wind blows.
- **Shape the product**. You will be laser-focused on the end goal. It's not about solving technical challenges but about the problem solved for the users.
- **Enable quality work**. You will:
  - Embody and foster the engineering culture, e.g. rigorousness, push for small single-purpose PRs, encourage peer reviews, create strong feedback loops between decision and outcome.
  - Empower the team to aim for high-quality outputs. By doing such it aims for the success of delivered solutions.
  - Push for consistency, follow what's going on in the other teams.
- **Keep technical debt in check**. You will make sure we can keep shipping features at a reasonable pace, align the team on "one way" of doing things and make sure engineers follow the conventions.

> - **Be a solver**. You will digs deep into arbitrarily complex problems and finds an appropriate path forward. Some focus on a given area for long periods, others bounce from hotspot to hotspot as guided by organizational leadership.

Reviewer: ```suggestion
- **Be a solver**. You will dig deep into arbitrarily complex problems and find an appropriate path forward. Some focus on a given area for long periods, others bounce from hotspot to hotspot as guided by organizational leadership.
```

- **Strong written and verbal communication skills**.
  As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
- **Ability to dive into complex problems**.
  You should be able to quickly assess, understand, and iterate upon aspects of our codebase.
- **Ready and willing to ask and answer questions**.
  If you're comfortable saying you're unsure, asking for help; but equally reaching out to assist others, you'll be an incredible addition to our team. We thrive because of continuous learning. First-time mistakes should be celebrated, not blamed.
- **Avoid monolithic deliverables**.
  You scope and stage your work into well-defined milestones to ship.
- **Past work with front-end infrastructure**.
  Perhaps you've created your company's design system, written a Babel plugin, or written complex React components.
  It would be great if you could address this in your cover letter!

### What it would be nice if you had, but isn't required

> - **You've contributed to a design system before**.

Author: I would say that for the Tech Lead role this may even be a requirement, not a "nice to have" thing.

Reviewer 2: Sold, it copy & pasted this too quickly.
